### DISCUSSION TITLE
Closing webcam putting when the putter is not selected.

### DISCUSSION THREAD

**Discussion by 1konig1** [springbok/MLM2PRO-GSPro-Connector#115] [Q&A]
Curious if it is possible to close the webcam putting app after one is finished putting, or the putter is no longer selected like matth's r10 bluetooth connecter.

I see the hide, minimize and send to back options on the putting system settings window, but I would like to close it completely.

I am working on configuring the floor projection shown on triple bogey golf's yt channel and have everything working through the obs screen, but due to the fact that the ball tracking exe not closing the putting screen can not leave the obs scene. 

Maybe I am missing a work around, but I was hoping to use the mlm2pro connector with my R10 and the webcam putting due to its nicer interface.

Thanks for any insight.

**Comment by springbok** [comment]
Ben from triple bogey golf have it working with the current options, see his YT channel https://www.youtube.com/channel/UCMPsAwfkRnRuB2FqQa8wDog you could contact him and ask him how he did it.
